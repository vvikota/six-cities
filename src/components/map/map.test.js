import React from 'react';
import renderer from 'react-test-renderer';
import Map from './map';

it(`Map is correctly renderer`, ()=> {
  const tree = renderer.create(<Map
    offers={[]}
  />).toJSON();

  expect(tree).toMatchSnapshot();
});
