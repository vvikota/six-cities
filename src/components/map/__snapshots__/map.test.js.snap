// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map is correctly renderer 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "90vh",
    }
  }
/>
`;
